lib/ui/login/login_screen.dart

```diff
@@ -0,0 +1,72 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:survey_flutter_ic/gen/assets.gen.dart';
+import 'package:survey_flutter_ic/ui/widget/input_field_widget.dart';
+import 'package:survey_flutter_ic/utils/dimension.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+
+class LoginScreen extends StatelessWidget {
+  const LoginScreen({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    SystemChrome.setSystemUIOverlayStyle(const SystemUiOverlayStyle(
+        statusBarColor: Colors.transparent, statusBarIconBrightness: Brightness.light, statusBarBrightness: Brightness.dark));
+    return Scaffold(
+        backgroundColor: Colors.transparent,
+        body: Container(
+          decoration: BoxDecoration(
+            image: DecorationImage(
+              image: AssetImage(Assets.images.bgLoginOverlay.path),
+              fit: BoxFit.cover,
+            ),
+          ),
+          child: Padding(
+            padding: const EdgeInsets.symmetric(horizontal: Dimensions.paddingLarge),
+            child: Center(
+              child: Column(
+                mainAxisAlignment: MainAxisAlignment.center,
+                children: [
+                  Padding(
+                      padding: const EdgeInsets.only(bottom: 109.0, left: 104.0, right: 104.0),
+                      child: Image.asset(
+                        Assets.images.icNimbleWhiteLogo.path,
+                      )),
+                  Padding(
+                      padding: const EdgeInsets.only(bottom: Dimensions.paddingMedium),
+                      child: InputFieldWidget(
+                        key: const Key('login_email'),
+                        textHint: AppLocalizations.of(context)!.loginEmail,
```

Comment:
As you can see, the localization will be called everywhere in the code; we'd better define an extension for the `BuildContext` to refer to it easily.

I want to call localization by this way:

```suggestion
                        textHint: context.localization.loginEmail,
```

with `localization` define like this:

```dart
extension BuildContextExtension on BuildContext {
    AppLocalizations get localization => AppLocalizations.of(this)!;
}
```

I think you will agree with me it looks much better, right?

Reply:
Really nice 👍